moveit_planners/pilz_industrial_motion_planner_testutils/include/pilz_industrial_motion_planner_testutils/goalconstraintsmsgconvertible.h

```diff
@@ -48,6 +48,7 @@ namespace pilz_industrial_motion_planner_testutils
 class GoalConstraintMsgConvertible
 {
 public:
+  virtual ~GoalConstraintMsgConvertible() = default;
```

Comment:
This will implicitly delete move semantics. Is that acceptable for these types?

Reply:
Valid point. I added default move/copy constructors and assignment operators.

Reply:
… and cleaned them up again where default constructors were not feasible.